### PR TITLE
Feature/update js handling

### PR DESCRIPTION
**Pull request by jake-kent**

### CODE REVIEW

**Comment by jschneier** [comment]
The test failure on this branch is a legit bug.

**Comment by jschneier** [comment]
In `base.html` we have `{% load js pipeline %}`. The first needs to be removed.

**Comment by jschneier** [comment]
While you're there can you remove the IE checks at the top. We don't use any of that.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!-->
 <html class="no-js">
-  <!--<![endif]-->
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>{%block title%} {%endblock title%}</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html]: There is another IE8 check lurking in there.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html [hunk 2]
```diff
@@ -1,19 +1,12 @@
...
-
-    {% load js pipeline %}
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html]: We still need `pipeline` (removing that is coming).

Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html]: That's injecting the `stylesheet` and `javascript` template tags.

File: {{cookiecutter.repo_name}}/requirements.txt [hunk 3]
```diff
@@ -48,8 +48,7 @@ pytz==2016.3
 easy-thumbnails==2.3
 
 #JS tools
-#django.js==0.8.1 # needs to be fixed up - using lightmatter version for now
-git+https://github.com/Lightmatter/django.js.git@24a9a41bf6aa7f454c4977b79059209434759b11#egg=django-js
+
 
 django-casper==0.0.4
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/requirements.txt]: Remove casper req.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py [hunk 4]
```diff
@@ -101,8 +101,6 @@ def get_env_setting(setting, default=None):
     'django_extensions',
     'model_utils',
     'pipeline',
-    'djangojs',
-    'casper',
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py]: There is a `djangojs` reference in this file.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html [hunk 5]
```diff
@@ -1,30 +1,20 @@
...
-  <!--<![endif]-->
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>{%block title%} {%endblock title%}</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    
     {% load js pipeline %}
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html]: Sorry, I wasn't clear. You can load the tags of multiple libraries in one statement. One is `js` - that is from `djangojs` and one is `pipeline` which is from `django-pipeline`.

So we need to change it from
`{% load js pipeline %}`
to
`{% load pipeline %}`

Comment (jake-kent) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html]: Ah I see

**Comment by jschneier** [comment]
LGTM

**Pull request merged**